workflows/templates/cilium.post-migrate.yaml

```diff
@@ -35,7 +35,27 @@ spec:
           - name: dryRun
       dag:
         tasks:
+        - name: restore-network-policies
+          templateRef:
+            name: ssh-template
+            template: shell-script
+          arguments:
+            parameters:
+              - name: dryRun
+                value: "{{inputs.parameters.dryRun}}"
+              - name: scriptContent
+                value: |
+                  cm_found=0
+                  kubectl -n argo get cm stash-netpol || cm_found=1
+                  if [ $cm_found -eq 1 ]; then
```

Comment:
Is this logic correct?  Shouldn't it be `&&` instead of `||`?  The way it is currently written `cm_found` will be set when the `kubectl` command fails.  But it can be simplified to one line:
```
    if kubectl -n argo get cm stash-netpol; then
```
Which can be simplified even more to just:
```
if kubectl -n argo get cm stash-netpol -o jsonpath='{.data.netpol\.yaml}' > /tmp/netpol-out.yaml; then
    kubectl apply -f /tmp/netpol-out.yaml
    kubectl -n argo delete cm stash-netpol
fi
```

Reply:
Fixed